chain/jsonrpc/src/lib.rs

```diff
@@ -1360,6 +1409,21 @@ fn get_cors(cors_allowed_origins: &[String]) -> Cors {
         .max_age(3600)
 }
 
+lazy_static_include::lazy_static_include_str! {
```

Comment:
:+1: this is indeed the intended use-case for lazy_static_include!

Reply:
ack